ChaosMod/Effects/db/Player/PlayerAdvice.cpp

```diff
@@ -38,7 +38,9 @@ std::vector<std::string> facts = {
 std::string msg;
 int scaleform = 0;
 
-static void _GetMessage(int rng) //Without underscore, it's defined somewhere idek.
+int debugIndex = 0;

```

Comment:
I guess, this shouldn't be here

Reply:
Thjat is when I was debugging it. One of the messages seemed to make the game crash.